riotfile.py

```diff
@@ -729,9 +729,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                             "~=7.6.0",
                             "~=7.8.0",
                             "~=7.10.0",
-                            # FIXME: Elasticsearch introduced a breaking change in 7.14
-                            # which makes it incompatible with previous major versions.
-                            # latest,
+                            "~=8.4.0",
```

Comment:
```suggestion
                            "~=8.4.0",
                            latest,
```

Reply:
This was integrated in the latest commit, thank you! 👍